Header header
bool rc_kill
bool rc_radio
uint8 power_good
float32 bat_v
float32 cpu_v
float32 cpu0_i
float32 cpu1_i
float32 modem_v
float32 modem_i
float32 sense_v
float32 lidar_i
float32 net_i